config/default.js

```diff
@@ -37,4 +37,13 @@ module.exports = {
     filename: './db.sqlite3',
     removeBeforeOpen: false,
   },
+  authentication: {
+    enabled: false,
+    defaultUser:  {              // DO NOT USE THIS IN PRODUCTION
+      email: 'user@example.com',
+      password: 'password',
+      name: 'Example User'
+    },
+    secret: 'top secret 51'      // DO NOT USE THIS IN PRODUCTION
+  }
 };
```

Comment:
By default, config assumes NODE_ENV is set to development, so we could put this in development.js and put a different set of defaults in production.js (and then we would arrange for NODE_ENV to be set to production for a real install)